### PR TITLE
updated datasets to 2021 and fixed terminology

### PR DESCRIPTION
**Pull request by gdaldegan**
updates Land Productivity TE, CHIRPS, Evapotranspiration datasets to include 2021 and fixes terminology reported on ticket #537

### CODE REVIEW

File: LDMP/gui/DlgCalculateLDNSummaryTableAdmin.ui [hunk 1]
```diff
@@ -345,7 +345,7 @@
                     </font>
                    </property>
                    <property name="text">
-                    <string>Trajectory (degradation)</string>
+                    <string>trend (degradation)</string>
```
Comment (azvoleff) [inline LDMP/gui/DlgCalculateLDNSummaryTableAdmin.ui:348]: @gabrieldaldegan - can you capitalize the "t" here?

File: LDMP/gui/DlgCalculateLDNSummaryTableAdmin.ui [hunk 2]
```diff
@@ -748,7 +748,7 @@
                     </font>
                    </property>
                    <property name="text">
-                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Trajectory (degradation)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;trend (degradation)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (azvoleff) [inline LDMP/gui/DlgCalculateLDNSummaryTableAdmin.ui:751]: Capital "t"

File: LDMP/gui/DlgCalculateProd.ui [hunk 3]
```diff
@@ -284,7 +284,7 @@
               <item>
                <widget class="QGroupBox" name="groupBox_traj">
                 <property name="title">
-                 <string>Trajectory (related to rate of change of  productivity over time)</string>
+                 <string>trend (related to rate of change of  productivity over time)</string>
```
Comment (azvoleff) [inline LDMP/gui/DlgCalculateProd.ui:287]: Capital "t"

File: LDMP/gui/DlgTimeseries.ui [hunk 4]
```diff
@@ -112,7 +112,7 @@
           </sizepolicy>
          </property>
          <property name="title">
-          <string>Trajectory indicator</string>
+          <string>trend indicator</string>
```
Comment (azvoleff) [inline LDMP/gui/DlgTimeseries.ui:115]: Capital "t"

File: LDMP/gui/DlgCalculateDroughtSummary.ui [hunk 5]
```diff
@@ -99,7 +99,7 @@
               </font>
              </property>
              <property name="title">
-              <string>Drought dataset (tiers 1 and 2)</string>
+              <string>Drought dataset (Hazard and Exposure)</string>
```
Comment (azvoleff) [inline LDMP/gui/DlgCalculateDroughtSummary.ui:102]: Lower case "h" and lower case "e"

File: LDMP/gui/DlgCalculateDroughtSummary.ui [hunk 6]
```diff
@@ -151,7 +151,7 @@
               </font>
              </property>
              <property name="title">
-              <string>Drought dataset for tier 3</string>
+              <string>Drought dataset for Vulnerability</string>
```
Comment (azvoleff) [inline LDMP/gui/DlgCalculateDroughtSummary.ui:154]: Lower case "v"

**Review by azvoleff — COMMENTED** [review]
@gdaldegan can you fix the casing (see comments)?

File: LDMP/gui/DlgCalculateDroughtSummary.ui (hunk 6 above)
Comment (gdaldegan) [reply LDMP/gui/DlgCalculateDroughtSummary.ui]: @azvoleff these fixes should be done now, thanks for spotting them

**Pull request merged**